tests/util/build_network_protocol_files.py

```diff
@@ -1,6 +1,9 @@
 # flake8: noqa
 
 import os
+import subprocess
+import sys
```

Comment:
I think you should remove the `sys` import (as highlighted by lgtm)

Reply:
I guess I'll have to dig into why nothing else complained about this, but yes, that is a remnant from when I tried to use `-m black`.

```suggestion
```

Reply:
Oh, the entire file got blanket `noqa`'ed.  Let's not do that either.